matchengine/internals/engine.py

```diff
@@ -613,7 +613,8 @@ def get_clinical_ids_from_sample_ids(self) -> Dict[ClinicalID, str]:
                     self._clinical_data.items()}
         else:
             return {clinical_id: clinical_data['SAMPLE_ID'] for clinical_id, clinical_data in
-                    self._clinical_data.items() if clinical_data['VITAL_STATUS'] == 'alive'}
+                    self._clinical_data.items() if (clinical_data['VITAL_STATUS'] is not None and clinical_data['VITAL_STATUS'].lower() == 'alive')}
```

Comment:
good catch, we should have that.

Reply:
The new ME version moves this logic around a bit. Now it ignores users with VITAL_STATUS "deceased," rather than including only users with VITAL_STATUS "alive" (i.e. everyone is alive by default).